skymodel.py

```diff
@@ -0,0 +1,208 @@
+import os
+import json
+import requests
+import urllib.error as ue
+
+import astropy.units as u
+from astropy.io import fits
+
+
+class ParameterError(Exception):
```

Comment:
Something more specific is preferred for the naming. Example:
```suggestion
class SkyCalcError(Exception):
```